### PR TITLE
EVEMonDevelopmentTeam.EVEMon version 4.1.3

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56000)

### CODE REVIEW

**Comment by superusercode** [comment]
I'm not sure where this is failing.

**Comment by superusercode** [comment]
@denelon Is it possible to get more verbose errors? I think the issue is that this app needs .NET to run/install but I'm not entirely confident on it.

**Comment by superusercode** [comment]
image
also, the site for smartscreen submissions seems to have changed. now it forces you to give it a malware or PUA detection name, which doesn't apply to the way winget submissions used to require it.

**Comment by denelon** [comment]
You can drill into the Azure DevOps build results and download artifacts. This is what I see:

```JSON
{
  "OverallResult": "Failure",
  "TestPlansResults": [
    {
      "TestPlan": "Validation-Installation-Error",
      "Result": "Failure",
      "Metadata": "",
      "ResultMetadata": {
        "Message": "-1978335226",
        "FieldsMetadata": [
          {
            "Field": "Exit Code",
            "Message": "-1978335226"
          },
          {
            "Field": "Architecture",
            "Message": "x86"
          },
          {
            "Field": "Scope"
          },
          {
            "Field": "Locale"
          }
        ]
      }
    }
  ],
  "AnalysisResults": []
}
```

**Comment by superusercode** [comment]
same error code "-1978335226" happened again. i can't find anything online about this, but this error code popped up a few times for other apps submitted to winget without resolution. the common denominator here is that it seems like app installers built with visual studio are affected? i don't have VS installed right now to check what kind of project type it is, but I believe this app still uses .net framework 4.0.

**Comment by superusercode** [comment]
image

File: manifests/e/EVEMonDevelopmentTeam/EVEMon/4.1.3/EVEMonDevelopmentTeam.EVEMon.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: EVEMonDevelopmentTeam.EVEMon
+PackageVersion: 4.1.3
```
Comment (ItzLevvie) [inline manifests/e/EVEMonDevelopmentTeam/EVEMon/4.1.3/EVEMonDevelopmentTeam.EVEMon.installer.yaml:5]:
```suggestion
PackageVersion: 4.1.3
InstallerSuccessCodes:
- 1223
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**